README.md

```diff
@@ -240,22 +240,527 @@ For complete documentation of the `mp-build` tool see the [MP Build Tool Documen
 
 ## Walkthroughs
 
-### Creating a New Management Pack (Cassandra-DB)
+### Creating a New Management Pack (Alibaba Cloud)
 <details><summary>
 This guide assumes you have already set up the SDK and know how to create a new project. 
-It walks you through the steps necessary to monitor an endpoint, using Cassandra DB as 
+It walks you through the steps necessary to monitor an endpoint, using Alibaba Cloud as 
 an example.</summary>
 TODO
 </details>
 
 ### Extending an Existing Management Pack (MySQL)
 <details><summary>
 This guide assumes you have already set up the SDK and know how to create a new project. 
-It walks you through the steps necessary to extend an existing Management Pact to add
+It walks you through the steps necessary to extend an existing Management Pack to add
 additional data, using the MySQL Management Pack as an example.</summary>
-TODO
-</details>
 
+Extending an existing management pack is similar to creating a new management pack, but 
+has some additional constraints. This section will create a management pack that adds 
+metrics to the existing MySQL management pack's database object. It assumes
+you have already installed the SDK and understand the tools and steps in the 'Get 
+Started' section. It also assumes that you have installed and configured the [MySQL
+management pack](https://customerconnect.vmware.com/downloads/details?downloadGroup=VRTVS_MP_MYSQL_810&productId=1051) 
+on a VMware Aria Operations instance in your local network.
+
+For the purposes of this walkthrough, we will be adding five metrics to the MySQL database
+object that show the total amount of lock waits and statistics about the time spent 
+waiting for those locks. This info can be found in MySQL in the table 
+`performance_schema.table_lock_waits_summary_by_table`.
+
+The first step is to run `mp-init` and create a new project. There are no restrictions,
+except that the adapter kind key cannot be used by another management pack that is
+installed on the same system. For example, we used the following to create the sample:
+
+```
+❯ mp-init
+Enter a directory to create the project in. This is the directory where adapter code, metadata, and
+content will reside. If the directory doesn't already exist, it will be created.
+Path: extended-mysql-mp
+Management pack display name: Extended MySQL MP
+Management pack adapter key: ExtendedMySQLMP
+Management pack description: Adds 'Lock Wait' metrics to MySQL Database objects
+Management pack vendor: VMware, Inc
+Enter a path to a EULA text file, or leave blank for no EULA: ../eula.txt
```

Comment:
where does ../eula.txt come from?

Reply:
We can remove it - it's not necessary for the walkthrough, but I used the same one we had in the apps-ops mp when I made the sample project